x-pack/platform/plugins/shared/maps/public/connected_components/toolbar_overlay/set_view_control/set_view_form.tsx

```diff
@@ -35,100 +43,70 @@ const COORDINATE_SYSTEM_OPTIONS = [
   },
 ];
 
-interface Props {
+interface SetViewFormProps {
```

Comment:
since Props is internal, they don't need to be named spaced. Can we just name them `Props`?